pkg/services/ldap/settings.go

```diff
@@ -118,7 +119,7 @@ func readConfig(configFile string) (*Config, error) {
 
 	logger.Info("LDAP enabled, reading config file", "file", configFile)
 
-	fileBytes, err := ioutil.ReadFile(configFile)
+	fileBytes, err := ioutil.ReadFile(filepath.Clean(configFile))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.